Draw/Elements/Value/ValueElementInSubflowId.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace ManyWho.Flow.SDK.Draw.Elements.Value
+{
+    [DataContract(Namespace = "http://www.manywho.com/api")]
+    public class ValueElementInSubflowIdAPI
+    {
+        public ValueElementInSubflowIdAPI()
+        {
+
+        }
+
+        public ValueElementInSubflowIdAPI(Guid id)
+        {
+            this.id = id.ToString();
+        }
+        
+        /// <summary>
+        /// The unique identifier for the Value to be referenced with this identifier.
+        /// </summary>
+        [DataMember]
+        public string id
```

Comment:
Lol, this is a new object - it can be a `Guid` straight up :)

Reply:
Oh yeah, I just copy pasted from the original type. Fixed that